x-pack/plugins/xpack_main/server/lib/feature_registry/feature_registry.ts

```diff
@@ -16,6 +16,7 @@ export interface FeaturePrivilegeDefinition {
   management?: {
     [sectionId: string]: string[];
   };
+  catalogue?: string[];
```

Comment:
How do we feel about this being defined within the privileges if we're using it for the "spaces driven feature controls", I believe we pulled the `navLinks` outside of the privileges for this reason previously.

Reply:
I agree having them defined within privileges feels strange in the context of spaces. We could make a similar argument for `management`, although `management` really does need privilege-specific handling, whereas `catalogue` does not (yet). 

I opted to put it here so that it was consistent with `management`, but if you'd rather see it consistent with `navLinks` instead, I can certainly do that. I'm worried about that not being flexible enough, if we get to the point where these catalogue entries need to become privilege-specific.